src/views/product-downloads-view/components/release-information/release-information.module.css

```diff
@@ -3,6 +3,18 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+.root {
+	& > * {
```

Comment:
Would you mind refactoring this to either flexbox or grid with the `gap` property?